### ISSUE TITLE
Compatibility with tasty

### ISSUE THREAD

**Issue by TimoMeijer** [chessai/hedgehog-classes#13]
As a common test runner, better integration with tasty and tasty-discover would be great.

**Comment by chessai** [comment]
Yeah, I totally agree. This should be a separate library (I'll probably put
it in  the same repo though).

**Comment by spacekitteh** [comment]
@chessai did anything ever come of this?

**Comment by chessai** [comment]
> @chessai did anything ever come of this?

No, I never did get around to it. PRs very much welcome.

**Comment by spacekitteh** [comment]
Damn, okay.
